### DISCUSSION TITLE
CERTIFICATION GitHub Foundations

### DISCUSSION THREAD

**Discussion by chouaib1123** [community/community#145439] [GitHub Education]
### Hi everyone,

i keep seeing no discount
![image](https://github.com/user-attachments/assets/2e1d8cc1-0180-4da7-ac01-764655e4ad73)
i refresh i go to private tab i go diff browser and always i see no discount

**Comment by khajavali1128** [comment]
Same with me sad😭

**Comment by md-ashraful-islam-ullas** [comment]
It's free now.

**Comment by khajavali1128** [comment]
Yes
I got it. Thank you

Khaja

**Comment by chouaib1123** [comment]
![image](https://github.com/user-attachments/assets/b6523c85-cfce-4268-9d52-05bfac1779b8)
always same issue

**Reply by md-ashraful-islam-ullas** [reply]
you are on right track. just click on "continue booking" to proceed. I guess you cannot choose specific date for exam (it wasn't for me either). However, you have to attend the exam within 60 days from the application date.

**Reply by chouaib1123** [reply]
thx you
